src/sage_docbuild/ext/sage_autodoc.py

```diff
@@ -56,8 +55,11 @@
 from sphinx.util.inspect import (evaluate_signature, getdoc, object_description, safe_getattr,
                                  stringify_signature)
 from sphinx.util.typing import OptionSpec, get_type_hints, restify
-from sphinx.util.typing import stringify as stringify_typehint
-
+try:
+    from sphinx.util.typing import stringify_annotation
+except ImportError:
+    from sphinx.util.typing import stringify as stringify_annotation
+    
 # ------------------------------------------------------------------
```

Comment:
There are invisible white spaces in L62. Remove them.

I would put a blank line before `try:`